spacewalk/certs-tools/spacewalk-certs-tools.changes

```diff
@@ -1,4 +1,5 @@
 - add GPG keys using apt-key on debian machines (bsc#1187998)
+- set key format to PEM when generating key for traditional clients push ssh (bsc#1189643)
```

Comment:
I think this should be on top and maybe break the line around 67th character

Reply:
Thanks Michael. I have pushed the changes